conftest.py

```diff
@@ -1,3 +1,4 @@
+import h5py
```

Comment:
I think `h5py` should be added to the test requirements?

Reply:
Yes, good catch. Is there a way to make the CI conda install be constructed starting from the `test` extra  + additional conda packages?

Reply:
Probably, but let's do that in a follow-up PR.